cmd/nerdctl/image/image_inspect_test.go

```diff
@@ -26,69 +26,52 @@ import (
 
 	"github.com/containerd/nerdctl/v2/pkg/inspecttypes/dockercompat"
 	"github.com/containerd/nerdctl/v2/pkg/testutil"
+	"github.com/containerd/nerdctl/v2/pkg/testutil/nerdtest"
+	"github.com/containerd/nerdctl/v2/pkg/testutil/test"
 )
 
-func TestImageInspectContainsSomeStuff(t *testing.T) {
```

Comment:
This turned out flaky as I was debugging this new code.
Rewriting against the new toolkit to fix that.